pkg/config/apiserver.go

```diff
@@ -13,7 +13,23 @@ type ApiServer struct {
 	// AdvertiseAddress in the loopback interface. Automatically computed.
 	SkipInterface bool `json:"-"`
 
+	AuditLog AuditLog `json:"auditLog"`
+
 	// The URL and Port of the API server cannot be changed by the user.
 	URL  string `json:"-"`
 	Port int    `json:"-"`
 }
+
+type AuditLog struct {
```

Comment:
I would expect adding this would update the default config (packaging/microshift/config.yaml) and configs in docs (docs/user/howto_config.md)